docs/src/modules/developing/pages/development-process-proto.adoc

```diff
@@ -48,6 +51,10 @@ TIP: For more information see xref:java-protobuf:actions-as-controller.adoc[Acti
 
 Services can interact asynchronously with other services and with external systems. Event Sourced Entities emit events to a journal, to which other services can subscribe. By configuring your own publish/subscribe (pub/sub) mechanism, any service can publish their own events and subscribe to events published by other services or external systems.
 
+=== Views
+
+A View provides a way to retrieve state from multiple Entities based on a query. You can query non-key data items. You can create views from Value Entity state, Event Sourced Entity events, and by subscribing to topics. For more information about writing views see xref:java:views.adoc[Implementing Views].
```

Comment:
`You can query non-key data items.` - I would remove this sentence.

Reply:
and there was a mistake on this text fragment, the link was pointing to the java page for Implementing Views, not the protobuf one. Fixed now.